salt/sshd/init.sls

```diff
@@ -1,9 +1,10 @@
 generate rsa key:
   cmd.run:
     - name: |
-        rm /etc/ssh/ssh_host_rsa_key &&
-        ssh-keygen -t rsa -b 4096 -N '' -f /etc/ssh/ssh_host_rsa_key
-    - unless: openssl rsa -text -noout -in /etc/ssh/ssh_host_rsa_key | grep "4096 bit"
+        cd /etc/ssh && rm ssh_host_*key*
+        ssh-keygen -t ed25519 -f ssh_host_ed25519_key < /dev/null
```

Comment:
I followed the URL already present in README.md:
https://stribika.github.io/2015/01/04/secure-secure-shell.html